backend/graphql/resolvers/testSessionResolvers.ts

```diff
@@ -30,75 +29,62 @@ const classService: IClassService = new ClassService(
   userService,
   testSessionService,
 );
-testSessionService.bindClassService(classService);
 
 const testSessionResolvers = {
   Query: {
-    testSession: async (
-      _parent: undefined,
-      { id }: { id: string },
-    ): Promise<TestSessionAndStudentResponseDTO> => {
-      const testSession: TestSessionResponseDTO =
-        await testSessionService.getTestSessionById(id);
-      const studentIdToDTO: Map<string, StudentResponseDTO> = new Map<
-        string,
-        StudentResponseDTO
-      >();
-      testSession.class.students.forEach((student) => {
-        studentIdToDTO.set(student.id, student);
-      });
-      return {
-        ...testSession,
-        results: testSession.results.map((result) => {
-          const studentDTO: StudentResponseDTO | undefined = studentIdToDTO.get(
-            result.student,
-          );
-          if (!studentDTO) {
-            throw new Error(
-              `Student id ${result.student} not found in class ${testSession.class.id}`,
-            );
-          }
-          return {
-            ...result,
-            student: studentDTO,
-          };
-        }),
-      };
-    },
-    testSessions: async (): Promise<TestSessionResponseDTO[]> => {
-      return testSessionService.getAllTestSessions();
-    },
-    testSessionByAccessCode: async (
+    testSession: (_parent: undefined, { id }: { id: string }) =>
```

Comment:
wondering why we removed the return values? is it just cause its verbose or there's actual code difference?

Reply:
Mainly verbose, yeah. Since we're returning the value of the service directly in basically all cases, it's just a duplicate type essentially